tools/walletextension/storage/sqlite.go

```diff
@@ -22,11 +22,15 @@ func NewSqliteDatabase(dbName string) (*SqliteDatabase, error) {
 		return nil, err
 	}
 
+	// todo (@ziga) - should we use binary format also for signature and text
 	_, err = db.Exec(`CREATE TABLE IF NOT EXISTS viewingkeys (
```

Comment:
I think we should have 2 tables:
1. Users (user_id, private_key)
2. Accounts(user_id, account_address, signature)

The signature is over a text in the right format, which you can recreate from the user_id, and account_address